diab_encounters/models.py

```
@@ -1,24 +1,24 @@
+    subjective = models.TextField(blank=True)
+    objective = models.JSONField(default=dict)
+    assessment = models.JSONField(default=dict)
+    plan = models.JSONField(default=dict)
+    created_by = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.PROTECT)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["patient", "occurred_at"]),
+            models.Index(fields=["created_by", "created_at"]),
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)